app/src/main/java/org/schabi/newpipe/player/NotificationUtil.java

```diff
@@ -121,6 +121,9 @@ synchronized void createNotificationIfNeededAndUpdate(final VideoPlayerImpl play
                 .setShowWhen(false)
                 .setSmallIcon(R.drawable.ic_newpipe_triangle_white)
                 .setColor(ContextCompat.getColor(player.context, R.color.gray))
```

Comment:
While you are at it, could you change the default color before media starts to play to a darker gray? `dark_background_color` should do
```suggestion
                .setColor(ContextCompat.getColor(player.context, R.color.dark_background_color))
```